src/graphql/data-sources/serlo.ts

```diff
@@ -248,6 +248,27 @@ export class SerloDataSource extends CacheableDataSource {
     return this.cacheAwareGet({ path: `/api/threads/${id}` })
   }
 
+  public async createThread(
+    payload: MutationCreateThreadArgs
+  ): Promise<ThreadsPayload> {
+    const firstCommentId = await this.customPost<number>({
+      path: `/api/create-thread/`,
+      body: payload,
+    })
```

Comment:
```suggestion
  public async createThread(
    payload: MutationCreateThreadArgs
  ): Promise<{ firstCommentId: number }> {
    return await this.customPost({
      path: `/api/create-thread/`,
      body: payload,
    })
```

The SerloDataSource should only have logic per public method for one endpoint -> other logic (like creating a ThreadPayload from the firstCommentId should be moved to the resolver).